builtin/logical/pki/path_tidy.go

```diff
@@ -115,6 +117,13 @@ var defaultTidyConfig = tidyConfig{
 	PublishMetrics:          false,
 }
 
+// assign the default value of revoked_safety_buffer to point to the default of safety_buffer
+func init() {
```

Comment:
If you leave it `nil`, this can be removed, which I think makes things cleaner :-)